nominees/datawinners.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "DataWinners",
+  "description": "DataWinners is a data collection application supporting multiple channels like sms, web and smartphone. Goal is to enable seamless data collection, reporting and visualization.",
+  "website": "http://datawinners.com/",
+  "license": [
+    {
+      "spdx": "Apache-2.0",
+      "licenseURL": "http://www.apache.org/licenses/LICENSE-2.0"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 4,
+      "evidenceURL": "https://allchildrenreading.org/innovator/human-network-international/"
+    },
+    {
+      "SDGNumber": 9,
+      "evidenceURL": "https://www.slideshare.net/SandraGubler1/human-network-international-its-services"
+    },
+    {
+      "SDGNumber": 10,
+      "evidenceURL": "https://www.slideshare.net/SandraGubler1/human-network-international-its-services"
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/mangroveorg/datawinners",
+  "organizations": [
+    {
+      "name": "Human Network Interaction (HNI)",
+      "website": "http://hni.org/",
```

Comment:
The website [link](http://hni.org/) shared appears to be a broken link. @rachita19082 could you kindly confirm this. Thanks.

Reply:
@nathanbaleeta This is the link on the [LinkedIn page of HNI](https://www.linkedin.com/company/human-network-international/) but it seems to be broken. I couldn't find any other link. Should I add the link to the LinkedIn page of HNI instead of website?

Reply:
You may use this for the [websiteURL](https://www.hni.ae/).

Reply:
@nathanbaleeta I came across this link but I am not sure if it is the same organisation. The [link](https://www.hni.ae/about-us/) shared by you refers to HNI Training and Coaching which is a training and consultancy organisation whereas the [HNI](https://www.linkedin.com/company/human-network-international/about/) to which this project belongs provides Information technology services, the logos are also different. Can you please confirm once if that is the right link?

Reply:
Good catch! @@rachita19082 well in that case please proceed with the LinkedIn [URL](https://www.linkedin.com/company/human-network-international/about/)

Reply:
@nathanbaleeta Updated in commit b4dabbb

Reply:
@nathanbaleeta @lacabra Please let me know if any other changes are required in this PR. Thanks.